cpp/ql/test/library-tests/ir/ir/raw_ir.expected

```diff
@@ -2882,55 +2882,50 @@ ir.c:
 #   25|     r25_1(glval<unknown>) = FunctionAddress[ExRaiseAccessViolation] : 
 #   25|     r25_2(glval<int>)     = VariableAddress[x]                      : 
 #   25|     r25_3(int)            = Load[x]                                 : &:r25_2, ~m?
-#   25|     v25_4(void)           = Call[ExRaiseAccessViolation]            : func:r25_1, 0:r25_3
-#   25|     mu25_5(unknown)       = ^CallSideEffect                         : ~m?
-#-----|   C++ Exception -> Block 6
+#-----|   Goto -> Block 2
+#-----|   SEH Exception -> Block 5
```

Comment:
The goto here is unexpected, as `ExRaiseAccessViolation` always raises an SEH exception.

Reply:
It is generating a goto and an SEH not for the call, but for the Load[x], all load instructions "MAY" throw an SEH exception, so you get a goto and an exception case.

Reply:
Note the call doesn't occur in this block at all anymore, it is now in Block 2, which is related to your next question.